README.md

```diff
@@ -0,0 +1,52 @@
+# cache-validation
+
+This action allows cache objects to be validated using MD5 checksums.
+
+[![Tests](https://github.com/techservicesillinois/awscli-login/actions/workflows/cache-validation.yml/badge.svg)](https://github.com/techservicesillinois/awscli-login/actions/workflows/cache-validation.yml)
+
```

Comment:
Needs an end-of-life date here based on Python 3.9 (Oct 2025) and NodeJs 14 (Jan 2023).

+ https://endoflife.date/python
+ https://endoflife.date/nodejs

So something like:

"As of the last update to this README, the expected End-of-Life and End-of-Support dates of this product are Jan 2023."